Apps/W1/ShopifyConnector/app/Order handeling/Codeunits/ShpfyOrderMapping.Codeunit.al

```diff
@@ -0,0 +1,172 @@
+/// <summary>
+/// Codeunit Shpfy Order Mapping (ID 30163).
+/// </summary>
+codeunit 30163 "Shpfy Order Mapping"
+{
+    Access = Internal;
+
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"